src/com/phronemophobic/membrane/term.clj

```diff
@@ -340,14 +340,34 @@
    :background      [1     1     1]
    :foreground      [0     0     0]})
 
-(def default-common-opts {:width 90
-                          :height 30
-                          :font-family :monospace
-                          :font-size 12
-                          :toolkit nil
-                          :color-scheme default-color-scheme})
+(def ^:private default-common-opts {:width 90
```

Comment:
generally, I think having the defaults be publicly available is useful.

Reply:
We should probably have `default-run-term-opts` and `default-screenshot-opts` then yeah?
`default-common-opts` is probably not as useful. Watcha think?